### ISSUE TITLE
overlap Math variable

### ISSUE THREAD

**Issue by philopon** [purescript/purescript-random#6]
Hi!

Purescript-random generate

```javascript
PS.Control_Monad_Eff_Random = (function () {
    "use strict";
    var Prelude = PS.Prelude;
    var Math = PS.Math; // <- overlap Math variable!
    var Control_Monad_Eff = PS.Control_Monad_Eff;

  function random() {
    return Math.random(); // <- error!!
  }
  ;
...
```

so runtime error occurred.

Thanks.

**Comment by paf31** [comment]
This is already fixed in the compiler. I just need to make a release.

**Comment by philopon** [comment]
I see. Thanks:smile:

**Comment by philopon** [comment]
I checked that this issue fixed on purescript-0.6.7. thanks!
